src/lib/components/toggle/stories/index.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { TickSVG, TimesSVG } from '../../../icons';
+import { ToggleButton } from '..';
+
+const meta: Meta<typeof ToggleButton> = {
+  title: 'Toggle',
+  component: ToggleButton,
+  argTypes: {
+    disabled: {
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ToggleButton>;
+
+export const Normal: Story = {
+  args: {
+    name: 'toggle',
+    defaultValue: false,
```

Comment:
put a void function to onChange event, only to see if its possible use this event on this componente